src/main/java/uk/gov/companieshouse/psc/delta/processor/PscDeltaProcessor.java

```diff
@@ -79,7 +80,7 @@ public void processDelta(Message<ChsDelta> chsDelta) {
      */
     public void processDelete(Message<ChsDelta> chsDelta) {
         final ChsDelta payload = chsDelta.getPayload();
-        final String logContext = payload.getContextId();
+        final String contextId = payload.getContextId();
```

Comment:
Can `notificationId` be declared on the same line as instantiation?

Reply:
done